src/SpecialFunctions.jl

```diff
@@ -57,11 +57,16 @@ export
     zeta,
     sinint,
     cosint,
-    lbinomial
+    lbinomial,
+    E1,
+    E₁,
+    En,
+    Eₙ
```

Comment:
On second thought, let's just export ASCII functions.    Maybe `expint` and `expint1`.

Reply:
Or just `expint(n, z)` and `expint(z)`

Reply:
with a docstring for `expint`